security.md

```diff
@@ -119,14 +119,14 @@ Add cert paths to your gemspec
 
 -------
 
-### Include checksum of released gems in your repository
-
-    require 'digest/sha2'
-    built_gem_path = 'pkg/gemname-version.gem'
-    checksum = Digest::SHA512.new.hexdigest(File.read(built_gem_path))
-    checksum_path = 'checksum/gemname-version.gem.sha512'
-    File.open(checksum_path, 'w' ) {|f| f.write(checksum) }
-    # add and commit 'checksum_path'
+### Include checksums of released gems in your repository
+
+Checksums can be created when you are ready to release a gem. Run:
+
+    rake build:checksum
+
+The checksums will be placed in the `checksums/` directory.  It is recommended to commit checksums so others can verify
+the authenticity of a release.
```

Comment:
I would migrate the previous instructions to use the builtin task and that's it. Officially recommending it should be a different discussion.

Reply:
Makes sense.  I'll update soon.

Reply:
Removed the "recommendation" implication.  Also improved the script to what I am currently using as I'm not sure when I'll get back around to the rake task, since in its current form it isn't useful to me at all, and I have a deep todo list.  I'll remove the script portion if preferred.